### ISSUE TITLE
Firefox related issue with dragging

### ISSUE THREAD

**Issue by bryanperris** [lukasoppermann/html5sortable#277]
I ran into a bug for Firefox 55.0.1, Ubuntu 64 bit.

On chrome (works): You can drag a tile and the placeholder moves around the list

On Firefox (bug): The placeholder shows up in one spot and stays there, therefore the draggable item is stuck in one place.  Sometimes when you click around like crazy on a draggable item, the sortable list works like its supposed to, and I can place the item anywhere I want.  I have found the same problem on the demo page as well (the grid one), so I know it doesn't have to do anything with my code, and this happens on 2 different Ubuntu machines.  I don't know about Windows or Mac.

I tracked down the culprit causing this issue.  Firefox is not firing dragenter and dragover events all the time during a drag, but for Chrome, these events fire all the time.  I created a hack so the function onDragOverEnter would get called a lot more when I am dragging.  Its a little whacky but acceptable, but however when adding an item to the list, the new item gets stuck in one place like before I made this hack.

So I changed the code like this to include the hack:
```
     ...
      _on(items.concat(sortableElement), 'dragover', onDragOverEnter)
      _on(items.concat(sortableElement), 'dragenter', onDragOverEnter)

      /* Hack for firefox to make drag callback get called more */
      if ($.browser.mozilla) {
        _on(items.concat(sortableElement), 'drag', onDragOverEnter)
      }
    ...
```

Hopefully a better solution can turn up.

Thanks!

**Comment by bdragoncore** [comment]
Odd, but I think Firefox 55.0.2 just somehow fixed this issue, sortable is now working 100% without the hack.

**Comment by lukasoppermann** [comment]
Okay, thanks. I am closing this issue, however, please let me know if it happens again.

**Comment by tolinho** [comment]
HI,
Running Firefox Quantum 57.0.1 (64-bit)
I can't sort the elements using this browser, not even in the example page:
https://lukasoppermann.github.io/html5sortable/index.html

**Comment by lukasoppermann** [comment]
Is that on windows or on linux? For me on mac osx it works fine.

**Comment by tolinho** [comment]
I'm Running Linux

**Comment by lukasoppermann** [comment]
Hmm, this is pretty weird. Do other plugins work for you?
I don't have linux, but if you could give some insights and maybe even a fix, that would be very awesome.

**Comment by tolinho** [comment]
Hi.
Yes other plugins work fine.
I also tested with no problem:
https://jqueryui.com/sortable
https://rubaxa.github.io/Sortable/

Later I found an other thread, my issue is exactly the same:
https://github.com/lukasoppermann/html5sortable/issues/285

I wish I had the skills to be able to help with a fix.

**Comment by LaurensRietveld** [comment]
I got the same issue here: sorting a grid list (including the one on the example page) does not work when using firefox on linux. Other browsers work fine
Firefox: 57.0.4
Linux: Ubuntu 17.10

The workaround provided in the OP fixes this issue. I suggest we re-open this issue, as this seems to be pretty reproducable

**Comment by lukasoppermann** [comment]
Okay @LaurensRietveld can you send a PR to fix this?

**Comment by LaurensRietveld** [comment]
I don't mind sending a PR, but haven't got a _proper_ solution though. What I forgot to say about this workaround is that it allows for re-arranging the tabs, but the placeholder 'flickrs', as if sortable can't properly detect which target element to put the placeholder. (let me know if this isnt clear enough, and I'll make a video).

**Comment by lukasoppermann** [comment]
A video would be very awesome, yes. 👍

**Comment by LaurensRietveld** [comment]
Caught in a catch22 here: want to record this issue in firefox quantum, but no firefox addon exists for recording the browser in quantum (only versions before).
Recording entire screen via os isn't possible as well, as I somehow record a black screen for all screen recorders ubuntu can give me. 

TLDR; can't really help you out with a video here...

**Comment by lukasoppermann** [comment]
Okay. If you find out how to fix it, please send a PR.

**Comment by lukasoppermann** [comment]
Hey @LaurensRietveld, I am sadly rather tight on timer plus I don't have a linux system easily available (nor do I have experience with it). If you could somehow assist with a FIX that would be neat.

**Comment by lukasoppermann** [comment]
Closed due to inactivity. If this issues still exists, please open a new issue, add browser version, test case, error log and any info that could help.
